src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
+@import "utils/variables";
+@import "utils/functions";
+@import "utils/mixins";
+
+@import "blocks/calendar";
+
 body {
```

Comment:
It's better to use a class instead of directly applying styles to the body element. Consider wrapping the content in a div with a specific class and apply styles to that class.